report/example.tex

```diff
@@ -106,6 +106,8 @@ \section{Introduction}
 %------------------------------------------------
 
 \section{Methods}
+In our analysis we applied the data-driven data science paradigm as to find new insights from the provided haircut tip data set.
+As compared to hypothesis-driven data science where a hypothesis/problem is formualted and then data is collected or found to help answer the hypothesis.
```

Comment:
Maybe highlight the fact that the data-driven approach is done on existing data? I know you have written "the provided haircut tip data set", but it may seem a bit vague to the reader if the data is pre-existing or newly collected.

Reply:
Probably too nitpicky, resolving this for now ;*